packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -213,6 +213,24 @@ def generic_location_header_during_redirect_is_adjusted_test(
         location_set,
         location_expected,
         ):
+    """Test if the Location header is rewriten by AR on redirect
+
+    This generic test issues a request to AR for a given path and verifies that
+    redirect has occurred with the `Location` header contents equal to
+    `location_expected` argument.
+
+    Arguments:
+        mocker (Mocker): instance of the Mocker class, used for controlling
+            upstream HTTP endpoint/mock
+        ar: Admin Router object, an instance of runner.(ee|open).Nginx
+        auth_header (dict): headers dict that contains JWT. The auth data it
```

Comment:
As a reader of this my aim is probably "What should I pass to this function to get a useful test".

I guess "A headers dict that contains a JWT containing invalid auth data".

If that's correct, it removes ambiguity about whether the data must or can be invalid.